src/main/java/me/gaminglounge/estats/listener/GetDamage.java

```diff
@@ -25,22 +27,32 @@ public class GetDamage implements Listener {
 
 
     @EventHandler
-    public void onDamagedbBy(EntityDamageByEntityEvent event){
+    public void onDamagedBy(EntityDamageByEntityEvent event){
 
-        //damageTest(event.getDamage(), event.getEntity().getType(), event.getDamager().getName(),event.getDamageSource().getDamageType());
-        //damageDisplay(event.getDamage(),event.getDamager());
         EStats.INSTANCE.entityName.add_(event.getEntity());
-        if(event.getDamager() instanceof Player & event.getEntity() instanceof LivingEntity){
+        if(event.getDamager() instanceof Player
+        || event.getDamager() instanceof Projectile
```

Comment:
Plese encapsulate the condition for `getDamager()` in parentheses for clear condition.
(Idk. what the java operator priority of `||` in comparison to `&`, but it could result in a wrong fallthrough)

Reply:
Fixed in https://github.com/GamingLounge-me/EStats/pull/6/commits/4b41a489959254b141a15a9baf330469bd785373

Reply:
Eee [my next review](https://github.com/GamingLounge-me/EStats/pull/6#pullrequestreview-2478758259)
or is this intendet?